package.json

```diff
@@ -3,7 +3,7 @@
   "scripts": {
     "dev": "svelte-kit dev",
     "build": "svelte-kit build",
-    "preview": "svelte-kit preview",
+    "preview": "svelte-kit preview --host",
```

Comment:
As mentioned in a previous comment, `--host` [looks scary](https://kit.svelte.dev/docs#command-line-interface-svelte-kit-dev). I'd prefer not using it.